app/Http/Controllers/AdminDash.php

```diff
@@ -952,6 +953,9 @@ public function showFeedback() {
     public function saveFeedback(Request $request, $id) {
         $feedback = Feedback::find($id);
         $feedback->controller_id = $request->controller_id;
+        if ($request->controller_id == '') {
+            $feedback->controller_id = $request->event_id;
+        }
```

Comment:
These 4 lines should be able to be combined into a 1 liner:

`$feedback->controller_id = $request->controller_id == '' ? $request->controller_id : $request->request_id;`

Reply:
Done